### ISSUE TITLE
Is it possible to have this running independently of Google Colab & drive?

### ISSUE THREAD

**Issue by shreks7** [TheLastBen/fast-stable-diffusion#142]
Just curious if we can run fast SD somehow on AWS or any other server without the need for Google services. Thank you.

**Comment by swcrazyfan** [comment]
Yes, you could run it on Paperspace Gradient for free. Or, pay $8/month and use their free GPU at that tier (I suggest the A4000). I've found them to be faster than what they give you in Colab. Plus, they won't randomly shutdown like Colab. You get 6 hours with a machine. Then, you can restart it. Plus ALL your files and work stay there. You only have to install the dependencies again.

I must say I don't have an easy notebook to share here for the GUI. However, I do have one for Dreambooth that I can share. Also, I can share a precompiled xformers for A400 GPU if you need it.

**Comment by shreks7** [comment]
> Yes, you could run it on Paperspace Gradient for free. Or, pay $8/month and use their free GPU at that tier (I suggest the A4000). I've found them to be faster than what they give you in Colab. Plus, they won't randomly shutdown like Colab. You get 6 hours with a machine. Then, you can restart it. Plus ALL your files and work stay there. You only have to install the dependencies again.
> 
> I must say I don't have an easy notebook to share here for the GUI. However, I do have one for Dreambooth that I can share. Also, I can share a precompiled xformers for A400 GPU if you need it.
> 
> If you try Paperspace, please use my referral code so we both get credit! Code: https://console.paperspace.com/signup?R=DWSNO4J

Thank you, I am going to try Paperspace. I think the Dreambooth one would be great for both trying locally and on the Paperspace. 

PS - Will use your referral code.

**Comment by cantaspinar** [comment]
@swcrazyfan Can you share your gradient notebook for Dreambooth?

**Comment by 0xdevalias** [comment]
See also:

- https://github.com/TheLastBen/fast-stable-diffusion/issues/80

**Comment by swcrazyfan** [comment]
> @swcrazyfan Can you share your gradient notebook for Dreambooth?

Yes, I can soon. I've been super busy so I haven't uploaded it to my repo or done a PR. However, I've been having issues with regularization images with the current version of diffusers, so you might need to pull an older version. I'll check and include that in the notebook.
